docs/source/migrating.rst

```diff
@@ -32,15 +32,14 @@ Non Wagtailtrans site
         # ...
     ]
 
-3. Add ``wagtailtrans.models.TranslatablePage`` to the existing ``wagtail.core.models.Page`` models.
+3. Replace ``wagtailtrans.models.TranslatablePage`` with the existing ``wagtail.core.models.Page`` models.
```

Comment:
I think this should be the other way around
```
Replace ``wagtail.core.models.Page`` with the existing ``wagtailtrans.models.TranslatablePage``  models.
```

Reply:
ah True, I'll fix that